### ISSUE TITLE
Permissions dask cluster

### ISSUE THREAD

**Issue by zonca** [zonca/pangeo-jetstream#1]
Now `dask-kubernetes` has support for RBAC, see https://github.com/pangeo-data/pangeo/pull/172.

However a Jupyter single-user pod of a user still has enough permissions to kill another user Jupyter or dask pod.
The solution would be namespaces, but it requires a large amount of work, see:
https://github.com/pangeo-data/pangeo/issues/135#issuecomment-384320753

@rsignell-usgs I suggest for now to go ahead and assume it is not a big deal and users won't try to delete other resources, is it ok?

If so I'll go ahead and try a complete deployment of Jupyterhub + dask-kubernetes (with each user being able to spawn clusters) + test reading zarr files.

**Comment by rsignell-usgs** [comment]
Yes, since pangeo hasn’t had problems yet, it seems reasonable. Also we will control users!
